app/services/state_file/send_post_deadline_reminder_service.rb

```diff
@@ -0,0 +1,39 @@
+module StateFile
+  class SendPostDeadlineReminderService
+    BATCH_SIZE = 10
+    HOURS_AGO = 24
+
+    def self.run
+      cutoff_time_ago = HOURS_AGO.hours.ago
+      intakes_to_notify = []
+
+      ApplicationRecord::STATE_INTAKE_CLASS_NAMES.each do |base_class|
+        class_object = base_class.constantize
+        intakes_to_notify += class_object.left_joins(:efile_submissions)
+                                         .where(efile_submissions: { id: nil })
+                                         .where("#{base_class.underscore.pluralize}.created_at < ?", cutoff_time_ago)
```

Comment:
From reading the requirements, I don't think there is a cutoff for how long ago the intake was created. Only a cutoff for if they already got the reminder.

Reply:
Good catch!